src/batch-client/index.ts

```diff
@@ -848,6 +848,12 @@ export class ZimbraBatchClient {
 			name: 'GetDataSources'
 		}).then(res => mapValuesDeep(res, coerceStringToBoolean));
 
+	public getDeviceStatus = () =>
+		this.jsonRequest({
+			name: 'GetDeviceStatus',
+			namespace: Namespace.Sync
+		}).then(res => get(res, 'device', []));
```

Comment:
get with third argument doesn't work in case of graphql data

Reply:
It is being used at other places as well. If its not harmful then let me keep this.

Reply:
yeah but it will not work when you get null as response

Reply:
Fixed.